package.yaml

```diff
@@ -14,16 +14,19 @@ extra-source-files:
 - ChangeLog.md
 
 dependencies:
-- base >=4 && <5
+- base >= 4.9.1 && <5
```

Comment:
Wow, you nailed the changelog, Travis, and hpack file changes. Color me impressed!